neo4j/templates/_helpers.tpl

```diff
@@ -1,4 +1,39 @@
 {{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "neo4j.defaultName" -}}
+    {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "neo4j.chart" -}}
+    {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
```

Comment:
are neo4j.Chart and neo4j.defaultName being used anywhere in the helm chart...i do not see any reference in the PR

Reply:
removed now